code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -391,11 +391,6 @@
 	take_damage(2 * X.xeno_caste.acid_spray_structure_damage, BURN, ACID)
 	return FALSE // not normal density flag
 
-/obj/vehicle/multitile/root/cm_armored/acid_spray_act(mob/living/carbon/xenomorph/X)
```

Comment:
do you need to readd this or something? Otherwise it does nothing.

Reply:
added for hitbox